app/models/user.rb

```diff
@@ -184,6 +184,9 @@ def self.get_authorization(provider, uid)
   BLOCKED_UIDS = ["example@bad.bad.server"]
 
   def self.from_omniauth(auth, current_user)
+    # reset twitter mastodon mapping cache upon human change of the user database
+    Rails.cache.delete('/twitter_mastodon_mapping')
```

Comment:
I think the cache should be devalidated (deleted), whenever a user is deleted or created. The position in the code is not obvious to me. I think it could fit here.